src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        Random colorRandom = new Random();
```

Comment:
Random should be class-level variable:

![image](https://github.com/mate-academy/jv-lottery/assets/140905715/d26605e3-e2e3-44ba-92a8-e6b1bde73c8d)